src/python/WMCore/JobStateMachine/ChangeState.py

```diff
@@ -581,7 +521,16 @@ def loadExtraJobInformation(self, jobs):
                 jobs[idx]["taskType"] = jobTask["type"]
                 jobs[idx]["jobType"] = jobTask["subtype"]
 
-    def completeCreatedJobsInformation(self, jobs, incrementRetry=False):
+    def completeCreatedJobsInformation(self, jobs, newstate, oldstate):
+        # If the new state is created it possible came from 3 locations:
```

Comment:
If I count correctly bellow the use cases listed are 4, or am I missing something?

Reply:
;)